### ISSUE TITLE
Error when generating code with swaggerCodeGen

### ISSUE THREAD

**Issue by angheladrianclaudiu** [unicredit/sbt-swagger-codegen#5]
Please see the below yaml definition.

```
swagger: '2.0'
info:
  version: '0.0.1-SNAPSHOT'
  title: Some test
  description: A service to test
  contact:
    name: Anghel Adrian
    email: adrian@emaple.com
    url: http://example.eu
basePath: /testSomeScripts
schemes:
  - http
consumes:
  - application/json
produces:
  - application/json
definitions:
  CallResult:
    required:
        - nameOfFunction
        - resultReturned
    properties:
        nameOfFunction:
          type: string
        resultReturned:
          type: string
  ErrorHttp:
    required:
      - code
      - message
    properties:
      code:
        type: integer
        format: int32
      message:
        type: string
      stackTrace:
        type: string
paths:
  /callScript/{scriptName}/{function}/{parametersList}:
    get:
      description: Call a function from a script
      operationId: callScript
      produces:
        - application/json
      parameters:
        - name : scriptName
          in: path
          required: true
          type: string
        - name: function
          in: path
          required: true
          type: string
        - name: parametersList
          in: path
          required: true
          type: array
          items:
            collectionFormat: csv
      responses:
        '200':
          description: function in the invoked script
          schema:
            $ref: '#/definitions/CallResult'
        default:
          description: unexpected error
          schema:
            $ref: '#/definitions/ErrorHttp'
```

This works correctly in swagger editor. But when I try to generate code using swaggerCodeGen task, I receive the following error:

<b>[main] ERROR io.swagger.models.properties.PropertyBuilder - no property for null, null 
[main] WARN io.swagger.util.PropertyDeserializer - no property from null, null, {ENUM=null, TITLE=null, DESCRIPTION=null, DEFAULT= TOR=null, MIN_ITEMS=null, MAX_ITEMS=null, MIN_PROPERTIES=null, MAX_PROPERTIES=null, MIN_LENGTH=null, MAX_LENGTH=null, MINIMUM=null MUM=null, EXCLUSIVE_MAXIMUM=null, UNIQUE_ITEMS=null, EXAMPLE=null, TYPE=null, FORMAT=null}</b>

Basically, I want to pass an array of objects, as described here: https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#itemsObject

What would be a workaround for this?

**Comment by andreaTP** [comment]
From a first analysis it seems that is missing the type of the items of the array, you can find an example here:
https://github.com/unicredit/sbt-swagger-codegen-examples/blob/master/src/main/swagger/petStore.yaml#L33

This i.e. works:

```
swagger: '2.0'
info:
  version: '0.0.1-SNAPSHOT'
  title: Some test
  description: A service to test
  contact:
    name: Anghel Adrian
    email: adrian@emaple.com
    url: http://example.eu
basePath: /testSomeScripts
schemes:
  - http
consumes:
  - application/json
produces:
  - application/json
definitions:
  CallResult:
    required:
        - nameOfFunction
        - resultReturned
    properties:
        nameOfFunction:
          type: string
        resultReturned:
          type: string
  ErrorHttp:
    required:
      - code
      - message
    properties:
      code:
        type: integer
        format: int32
      message:
        type: string
      stackTrace:
        type: string
paths:
  /callScript/{scriptName}/{function}/{parametersList}:
    get:
      description: Call a function from a script
      operationId: callScript
      produces:
        - application/json
      parameters:
        - name : scriptName
          in: path
          required: true
          type: string
        - name: function
          in: path
          required: true
          type: string
        - name: parametersList
          in: path
          required: true
          type: array
          items:
            type: string
          collectionFormat: csv
      responses:
        '200':
          description: function in the invoked script
          schema:
            $ref: '#/definitions/CallResult'
        default:
          description: unexpected error
          schema:
            $ref: '#/definitions/ErrorHttp'
```
